main.js

```diff
@@ -33,6 +34,7 @@ var currentActivatedCategory = "";
 var currentActivity;
 var startTime;
 var updateTimer;
+var savedActivities = [];
```

Comment:
Is this being used anywhere?

Reply:
Good catch! Wasn't being used, but is now.